src/functions-and-arrays.js

```diff
@@ -1,20 +1,40 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
+function maxOfTwoNumbers(num1, num2) {
+  return Math.max(num1, num2);
+}
+maxOfTwoNumbers(43, 24)
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
-
+function findLongestWord(words) {
+  if (words.length === 0){
+    return null;
+  }
```

Comment:
For shorter syntax, if you return only on line after if statement, you could write it like this:
```
if (words.length === 0) return null;
```